### PR TITLE
Release notes for WKO 4.0.5

### PR DESCRIPTION
**Pull request by rjeberhard**

### CODE REVIEW

File: documentation/4.0/content/release-notes.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ draft: false
...
+| February 21, 2023  | v4.0.5   | Resolved.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```
Comment (doxiao) [inline documentation/4.0/content/release-notes.md:13]: A summary of resolved issues is missing. Maybe this is intentional, but just want to make sure.

**Review by doxiao — COMMENTED** [review]
LGTM. Just had one minor comment.

**Review by ankedia — APPROVED** [review]

**Review by jshum2479 — APPROVED** [review]

File: documentation/4.0/content/release-notes.md (hunk 1 above)
Comment (rjeberhard) [reply documentation/4.0/content/release-notes.md]: Thanks! Yes, you're right, this was supposed to be a "todo" marker so that I would write this message.

**Review by doxiao — APPROVED** [review]

File: documentation/4.0/content/release-notes.md [hunk 2]
```diff
@@ -65,6 +66,26 @@ draft: false
 
 ### Change log
 
+#### Operator 4.0.5
+
+* Updated the Fluentd integration to make the Elasticsearch credentials optional and to allow the specification of the container command and arguments.
+* Resolved an issue related to growth in the number of WebLogic sessions related to the monitoring exporter and health checks.
+* Resolved [issue #3865](https://github.com/oracle/weblogic-kubernetes-operator/issues/3865) related to decorating the name of the ConfigMap for Fluentd integration ([#3883](https://github.com/oracle/weblogic-kubernetes-operator/pull/3883)).
+* Updated the generation of pods and containers for [security best practices](https://oracle.github.io/weblogic-kubernetes-operator/security/pod-and-container/).
```
Comment (rosemarymarano) [inline documentation/4.0/content/release-notes.md:74]: -> I think that the x-ref should be to /security/**domain-security**/pod-and-container/

**Review by rosemarymarano — APPROVED** [review]
One minor fix.

**Pull request merged**